pyreisejl/utility/launchers.py

```diff
@@ -61,45 +81,64 @@ def __init__(self, start_date, end_date, interval, input_dir):
         self.n_interval = int(ts_range / interval)
         self.input_dir = input_dir
         print("Validation complete!")
+        # These parameters are not validated
+        self.execute_dir = execute_dir
+        self.threads = threads
+        self.solver_kwargs = solver_kwargs
+        self.julia_env = julia_env
+        self.num_segments = num_segments
 
     def _print_settings(self):
         print("Launching scenario with parameters:")
         print(
             {
+                "num_segments": self.num_segments,
                 "interval": self.interval,
                 "n_interval": self.n_interval,
                 "start_index": self.start_index,
                 "input_dir": self.input_dir,
                 "execute_dir": self.execute_dir,
                 "threads": self.threads,
+                "julia_env": self.julia_env,
+                "solver_kwargs": self.solver_kwargs,
             }
         )
 
+    def init_julia(self, imports=["REISE"]):
+        """Initialize a Julia session in the specified environment and import.
+
+        :param list imports: julia packages to import.
+        :return: (*tuple*) -- imported names.
+        """
+        api = LibJulia.load()
+        julia_command_line_options = ["--compiled-modules=no"]
+        if self.julia_env is not None:
+            julia_command_line_options += [f"--project={self.julia_env}"]
+        api.init_julia(julia_command_line_options)
+
+        return tuple([importlib.import_module(f"julia.{i}") for i in imports])
+
+    def parse_and_return_runtime(self, start, end):
```

Comment:
I might call this just `parse_runtime` since the return is kinda implied

Reply:
Done